compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/ProjectDirConstants.java

```diff
@@ -33,7 +33,7 @@ private ProjectDirConstants() {
     public static final String BLANG_COMPILED_PKG_EXT = ".zip";
 
     public static final String MANIFEST_FILE_NAME = "Ballerina.toml";
-    public static final String PACKAGE_MD_FILE_NAME = "Package.md";
+    public static final String MODULE_MD_FILE_NAME = "Module.md";
```

Comment:
I think this should be fine because AFAIK, when the new version released old packages, are being outdated with the new balo version and have to release a new version of the package/module

Reply:
Yes we have decided to change 'Package.md; to 'Module.md'. Users with old packages have to rename Package.md to Module.md

Reply:
Ah got it! thanks